pkg/issuer/venafi/client/instrumentedvenaficlient.go

```diff
@@ -44,46 +48,41 @@ func newInstumentedConnector(conn connector, metrics *metrics.Metrics, log logr.
 }
 
 func (ic instrumentedConnector) ReadZoneConfiguration() (*endpoint.ZoneConfiguration, error) {
```

Comment:
This change changes the behavior of the `ReadZoneConfiguration` function.
The duration is supposed to be the time since the start of the function call.

Reply:
Thanks, I fixed it. Also I would like to ask: If there a specific reason why Venafi client located in pkg/issuer/ folder and not in internal/ as Vault one ? I thought move it alongside with vault client, but maybe there are a specific reason for it.

Reply:
I don't think there is a specific reason (however, I often find that I'm wrong when I try to move things and figure out that they are imported in a lot of different places).